consensus/cryptarchia-engine/src/config.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{Epoch, Slot};
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct TimeConfig {
+    // How long a slot lasts in seconds
+    pub slot_duration: u64,
+    // Start of the first epoch, in unix timestamp second precision
+    pub chain_start_time: u64,
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct Config {
+    // Depth of forks that trigger the density rule
+    pub k: u8,
```

Comment:
k is actually the `Security Parameter`. It controls how the block depth before a block is considered immutable.